ajax/libs/highmaps/package.json

```diff
@@ -16,8 +16,36 @@
       "web": "http://highsoft.com/"
     }
   ],
-  "license": {
-    "type": "Creative Commons Attribution-NonCommercial 3.0 License.",
-    "url": "http://creativecommons.org/licenses/by-nc/3.0/"
+  "license": "CC-BY-NC-3.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/highslide-software/highmaps-release.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/highslide-software/highmaps-release.git",
+    "basePath": "",
+    "files": [
```

Comment:
It's a release repo just like the dist folder, let's use `**/*.+(css|js|png)` here